fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -964,6 +968,22 @@ cleanTabletSchedQClause
     : CLEAN TABLET SCHEDULER QUEUE
     ;
 
+decommissionDiskClause
+    : DECOMMISSION DISK string (',' string)*
+    ;
+
+cancelDecommissionDiskClause
+    : CANCEL DECOMMISSION DISK string (',' string)*
+    ;
+
+disableDiskClause
+    : DISABLE DISK string (',' string)*
```

Comment:
Fot not implemented sql interface, we should return an unsupported error message when user executes that sql.(In EE version, we don't.)

Reply:
done